src/briefcase/integrations/android_sdk.py

```diff
@@ -327,7 +606,14 @@ def force_stop_app(self, package):
         # In my testing, `force-stop` exits with status code 0 (success) so long
         # as you pass a package name, even if the package does not exist, or the
         # package is not running.
-        self.run("shell", "am", "force-stop", package)
+        try:
+            self.run("shell", "am", "force-stop", package)
+        except subprocess.CalledProcessError:
+            raise BriefcaseCommandError(
```

Comment:
Given the comments above about `force-stop` exiting with 0 in all cases, I'm curious if you triggered this error. If not, then maybe we're not catching the error condition accurately.

The only condition in which I could imagine this happening is if the device has gone away. In that case, I'd prefer to show an error message about the device not being connected.

Reply:
Again - it's a safety mechanism, combined with the refactoring of the exception handling on `ADB.run()`.